src/http/representation/RepresentationMetadata.ts

```diff
@@ -21,6 +21,11 @@ export function isRepresentationMetadata(object: any): object is RepresentationM
   return typeof object?.setMetadata === 'function';
 }
 
+// Caches named node conversions
+const cachedNamedNodes: Record<string, NamedNode> = {
+  contentType: CONTENT_TYPE_TERM,
```

Comment:
This line can go. I don't think there is any place in the code where we actually use the shortcut like this.

Reply:
Meaning the initialisation with contentType? Because the variable is used in `toCachedNamedNode`

Reply:
Yes the initialisation. `cachedNamedNodes` can stay.

Reply:
Removing that initialisation broke `ContentTypeReplacer.test` and `ConstantConverter.test` 🤔 
![image](https://user-images.githubusercontent.com/4209558/163349802-afd38713-3587-4341-8bcc-b86202cf5549.png)

Reply:
Guess I was wrong and this is used in those 2 specific places, which happen to be 2 classes I didn't write and explains why I didn't think it was used anywhere 😄. It's only in the tests though as far as I can see, so I would update those so they also use the `CONTENT_TYPE` constant instead for consistency with all the other places.